sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -418,6 +417,7 @@ def test_check_estimator():
     check_estimator(LogisticRegression(C=0.01))
     check_estimator(MultiTaskElasticNet)
     check_estimator(MultiTaskElasticNet())
+    check_estimator(RandomForestClassifier())
```

Comment:
Not sure what this is meant to ensure.

Reply:
good question, I kept it from rebase but don't see either why it was added